karapace/protobuf/proto_file_element.py

```diff
@@ -29,18 +32,23 @@ def _collect_dependencies_types(compare_types: CompareTypes, dependencies: Optio
         _collect_dependencies_types(compare_types, sub_deps, is_self)
 
 
+# todo: transform those types into serious object with parsing that filters out invalid type name or invalid package name
+TypeName = str
```

Comment:
suggestion: Can we turn these into NewType now already? Bare type aliases like this are a bit devious as they give an impression of stricter type safety where there is none.

Reply:
yeah, small effort, make sense to push for it